ts/quill/emoji/completion.tsx

```diff
@@ -120,11 +120,12 @@ export class EmojiCompletion {
     if (!range) return PASS_THROUGH;
 
     const [blot, index] = this.quill.getLeaf(range.index);
+    blot.text = blot.text?.toLowerCase()
```

Comment:
1. This line needs a semicolon at the end
2. This mutates the blot. Are there any side-effects of doing that? (I assume not but want to make sure we've checked.)

Reply:
2. From a quick look, only `blot.text` is used after this change has been made, so that should be ok. But that also means that we might be better off just passing `blob.text` (changed to lower case) instead of passing the whole blot object. I'll create a new PR with this alternative, so that you can choose what you prefer.

Reply:
See https://github.com/signalapp/Signal-Desktop/pull/5186 for the other implementation. If one of these two is good enough just close the other one.